roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | replace(':','=') | replace('{', '') | replace('}', '') | replace(' ', '') | replace(\"u\'\", '') | replace(\"\'\", '') }}"
```

Comment:
@kwoodson @jcantrill I am very unhappy with what I did here but it's the only thing that worked.
IIUC, nodeselector labels for deployment are formatted in dictionary mode, while in namespace the equality-based definition is used (https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
So I had to convert it somehow, and I didn't want to write an entire module for that.

Do you have a better proposal on how to accomplish this?
@kwoodson does it seem logical that the oc_project module will also accept dictionary based labeling and convert it?